packages/ui/package.json

```diff
@@ -11,7 +11,7 @@
   },
   "scripts": {
     "build": "npm run build:lib && npm run build:es",
-    "build:lib": "rm -rf dist && NODE_ENV=production BABEL_ENV=lib babel lib --out-dir dist --copy-files",
+    "build:lib": "rm -rf dist",
     "build:es": "rm -rf es && NODE_ENV=production BABEL_ENV=es babel lib --out-dir es --copy-files",
```

Comment:
if only `es` version left, just change this script to `build` and remove above.

Reply:
also need to remove `files: ["dist"]`, just keep `"es"`

Reply:
remove yet let us keep in mind to remove local `dist` since we dont want to upload it anymore